Mathlib/LinearAlgebra/BilinearForm/Basic.lean

```diff
@@ -343,36 +335,28 @@ variable (R₂)
 /-- The flip of a bilinear form, obtained by exchanging the left and right arguments. This is a
 less structured version of the equiv which applies to general (noncommutative) rings `R` with a
 distinguished commutative subring `R₂`; over a commutative ring use `flip`. -/
```

Comment:
This docstring is now out of date

Reply:
Fixed. (N.B. `flipHom` and `flip` are now the same thing.)